test/counterTest.js

```diff
@@ -80,13 +90,6 @@ describe('counter', () => {
 				expect(fn).toThrowErrorMatchingSnapshot();
 			});
 
-			it('should throw error if label lengths does not match', () => {
```

Comment:
it should still throw an error unless the argument is a plain error

Reply:
I removed these lines because this test is a complete duplicate of the one above it